content/docs/reference/pkg/nodejs/pulumi/kubernetes/_index.md

```diff
@@ -6,23 +6,6 @@ linktitle: "@pulumi/kubernetes"
 <!-- WARNING: this page was generated by a tool. Do not edit it by hand. -->
 <!-- To change it, please see https://github.com/pulumi/docs/tree/master/tools/tscdocgen. -->
 
-
```

Comment:
@lblackstone, looks like this is due to https://github.com/pulumi/pulumi-kubernetes/pull/784

Reply:
Will revert this change.

Reply:
https://github.com/pulumi/pulumi-kubernetes/pull/811